assets/contracts/mocks/ERC721Mock.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: CC0-1.0
+
+pragma solidity ^0.8.16;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+
+/**
+ * @title ERC721Mock
+ * Used for tests
+ */
+contract ERC721Mock is ERC721, Emotable {
```

Comment:
This is surely broken, it's inheriting a non imported contract.
Which doesn't need to inherit anymore anyway

Reply:
It seems that GitHub has once again not showed all of the changes 🤔  This was already removed in the following commit.